CONTRIBUTING.md

```diff
@@ -11,18 +11,31 @@ You can build and run virtual users on MacOS, Windows or Linux. You'll need JDK
 
 ## Submitting changes
  
-Pull requests, issues and comments are welcome. For pull requests:
+Pull requests, issues and comments are welcome. Below checklist can help you to go smoothly through the review.
+Please remember that common sense applies here. 
+
+### Pull Request Readiness readiness checklist
+
+  * [ ] Includes tests to exercise the new behavior and bugs.
```

Comment:
Are these checkboxes supposed to be ticked? [They're not tickable](https://github.com/wyrzyk/jira-software-actions/blob/6a52e6d644db95c855b621d4595a6b4fff50c395/CONTRIBUTING.md#pull-request-readiness-readiness-checklist).
If we want to create a literal PR checklist, we can use a [PR template](https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository).